dmutils/dates.py

```diff
@@ -0,0 +1,54 @@
+from six import string_types
+from datetime import datetime, timedelta
+from workdays import workday
+
+
+def get_publishing_dates(brief):
+    APPLICATION_OPEN_DAYS = {'1 week': 7, '2 weeks': 14}
+    QUESTIONS_OPEN_DAYS = {'1 week': 2, '2 weeks': 5}
+    dates = {}
+
+    if brief_is_from_api_(brief):
+        dates['today'] = get_start_date_from_brief_api(brief)
```

Comment:
Would it make sense to call it `dates['published_date']` or something similar rather than `dates['today']`? If this method is run on a brief not on the day of publishing then you are not actually given todays date.